src/OmniSharp.Roslyn.CSharp/Services/DocumentationConverter.cs

```diff
@@ -126,5 +128,131 @@ private static string GetCref(string cref)
+                                    currentSectionBuilder.Append(GetCref(xml["cref"]));
+                                    break;
+                                case "paramref":
+                                    currentSectionBuilder.Append(xml["name"]);
+                                    currentSectionBuilder.Append(" ");
+                                    break;
+                                case "param":
+                                    StringBuilder paramInstance = new StringBuilder();
+                                    paramInstance.Append(TrimMultiLineString(xml["name"], lineEnding));
+                                    paramInstance.Append(": ");
+                                    currentSectionBuilder = paramInstance;
+                                    paramElements.Add(paramInstance);
+                                    break;
+                                case "typeparamref":
+                                    currentSectionBuilder.Append(xml["name"]);
+                                    currentSectionBuilder.Append(" ");
+                                    break;
+                                case "typeparam":
+                                    StringBuilder typeParamInstance = new StringBuilder();
+                                    typeParamInstance.Append(TrimMultiLineString(xml["name"], lineEnding));
+                                    typeParamInstance.Append(": ");
+                                    currentSectionBuilder = typeParamInstance;
+                                    typeParamElements.Add(typeParamInstance);
+                                    break;
+                                case "value":
+                                    valueText.Append("Value: ");
+                                    currentSectionBuilder = valueText;
+                                    break;
+                                case "br":
+                                case "para":
+                                    currentSectionBuilder.Append(lineEnding);
+                                    break;
+                            }
+                        }
+                        else if (xml.NodeType == XmlNodeType.Text && currentSectionBuilder != null)
+                        {
+                            if (elementName == "code")
+                            {
+                                currentSectionBuilder.Append(xml.Value);
+                            }
+                            else
+                            {
+                                currentSectionBuilder.Append(TrimMultiLineString(xml.Value, lineEnding));
+                            }
+                        }
+                    } while (xml.Read());
+                }
+                catch (Exception)
+                {
+                    return null;
+                }
+
+                docComment.RemarksText = remarksText.ToString();
```

Comment:
How do you feel about adding a constructor to docComment that takes all of these, rather than having to remember to set all of the properties? (Maybe makes it easier to not forget a section if we refactor).